tests/publiccloud/az_accelerated_net.pm

```diff
@@ -96,11 +105,11 @@ test on the client side. The test runs TEST_TIME seconds.
 =cut
 sub run_test {
     my ($self, $client, $server) = @_;
-    record_info('server', 'Start IPERF in server' . $server->public_ip);
-    $server->run_ssh_command(cmd => 'nohup iperf -s -D &', no_quote => 1);
+    record_info('server', 'Start IPERF in server ' . $server->public_ip);
+    $server->run_ssh_command(cmd => 'nohup iperf3 -s -D &', no_quote => 1);
     sleep 60;    # Wait 60 seconds so that the server starts up safely and the clinet can connect to it
     record_info('client', 'Start IPERF in client');
-    my $output = $client->run_ssh_command(cmd => 'iperf -t ' . get_required_var('TEST_TIME') . ' -c ' . $server->public_ip);
+    my $output = $client->run_ssh_command(cmd => 'iperf3 -t ' . get_required_var('TEST_TIME') . ' -c ' . $server->{private_ip});
```

Comment:
do we need to define some `timeout => `  in case TEST_TIME is higher than the default timeout in the script ssh call?

Reply:
So let's say `timeout => TEST_TIME * 5` ?

Reply:
or simply `TEST_TIME + 5`?

Reply:
would that be enough? idk, what are reasonable values for the TEST_TIME variable. By default the doc claims 10.

Reply:
Not sure, not sure if iperf3 needs some more time after TEST_TIME... I leave it up to you to decide, but let's make sure that at least is higher than TEST_TIME should be safe :)

Reply:
Let's go for multiply of 3. Can be changed later as well